cli/install/install.go

```diff
@@ -899,6 +899,19 @@ func installTarantool(binDir string, incDir string, installCtx InstallCtx,
 	return nil
 }
 
+func dirsAreWriteable(dirs []string) bool {
+	for _, dir := range dirs {
+		tempFileName := filepath.Join(dir, "tempfile")
+		err := os.WriteFile(tempFileName, []byte{}, 0)
```

Comment:
Why don't you use `os.Stat()`?

Reply:
I use the unix syscall `access` for finding out if the current user has write access.